.github/ISSUE_TEMPLATE/installation_issue.yaml

```diff
@@ -0,0 +1,67 @@
+name: Installation Issue
+description: Report issues installing the pandas library on your system
+title: "BUILD: "
+labels: [Build, Needs Triage]
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      options:
+        - label: >
+            I have read the [installation guide](https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html#installing-pandas).
+          required: true
+  - type: textarea
+    id: operating-system
+    attributes:
+      label: Operating System
+      description: >
+        Please provide the operating system you are running.
+      placeholder: >
+        Windows 10
+
+        macOS 11 Big Sur
+
+        Ubuntu 18.04 LTS
+    validations:
+      required: true
+  - type: textarea
+    id: platform
+    attributes:
+      label: Platform
+      description: >
+        Please provide the hardware platform you are running.
+      placeholder: >
+        x86-64
+
+        ARM64 (M1)
+    validations:
+      required: true
+  - type: dropdown
+    id: method
+    attributes:
+      label: Installation Method
+      description: >
+        Please provide how you tried to install pandas.
+      options:
+        - pip install
+        - conda install
+        - apt-get install
+        - Built from source
+        - Other
```

Comment:
Not sure where best we could put this - but it might be helpful to suggest that the issue should be reproducible from a _clean_ environment. Seems like lots of build issues are caused by issues that go away in a fresh env